frontend/javascripts/oxalis/view/left-border-tabs/modals/add_volume_layer_modal.tsx

```diff
@@ -34,6 +34,13 @@ export default function AddVolumeLayerModal({
   const datasetResolutionInfo = getDatasetResolutionInfo(dataset);
   const [resolutionIndices, setResolutionIndices] = useState([0, 10000]);
 
+  useEffect(() => {
```

Comment:
I think this is redundant here, as the same code does already exist within `RestrictResolutionSlider`. Can you remove it here and test that the UI still works when not changing the slider?

Reply:
Got it. Yes, works as expected :)